### PR TITLE
#128_create_search_for_transport

### PR DESCRIPTION
**Pull request by swem-st**

### CODE REVIEW

File: RabbitMQConfig/ConfigureServicesMassTransit.cs [hunk 1]
```diff
@@ -41,8 +41,8 @@ public static class ConfigureServicesMassTransit
                     {
                         cfg.Host($"rabbitmq://{url}/{host}", configurator =>
                         {
-                            configurator.Username(rabbitMQSection.GetValue<string>("UserName"));
-                            configurator.Password(rabbitMQSection.GetValue<string>("Password"));
+                            configurator.Username("easy_ca_user");
+                            configurator.Password("Qwerty211@");
```
Comment (SashaKorniichuk) [inline RabbitMQConfig/ConfigureServicesMassTransit.cs]: move username and password to Secret file

File: RabbitMQConfig/Models/Requests/TransportRequestModelRabbitMQ.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RabbitMQConfig.Models.Requests
+{
+    public class TransportRequestModelRabbitMQ
+    {
+        public string Filter { get; set; }
+
```
Comment (SashaKorniichuk) [inline RabbitMQConfig/Models/Requests/TransportRequestModelRabbitMQ.cs]: delete empty line

File: RabbitMQConfig/Models/Responses/TransportResponseModelRabbitMQ.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        public string Id { get; set; }
+        public string ProducedBy { get; set; }
+        public string Model { get; set; }
+        public string CategoryName { get; set; }
+        public string VINCode { get; set; }
+        public string CarPlate { get; set; }
+        public string Color { get; set; }
+        public int YearOfProduction { get; set; }
+        public InsuaranceModelRabbitMQ InsuaranceNumber { get; set; }    
+
```
Comment (SashaKorniichuk) [inline RabbitMQConfig/Models/Responses/TransportResponseModelRabbitMQ.cs]: delete empty line

File: SearchMicroservice.Domain/Services/SearchService.cs [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+namespace SearchMicroservice.Domain.Services
+{
+    public class SearchService : ISearchService
+    {
+        private readonly IClientFactory _clientFactory;
+        public SearchService(IClientFactory clientFactory)
+        {
+            _clientFactory = clientFactory;
+        }
```
Comment (SashaKorniichuk) [inline SearchMicroservice.Domain/Services/SearchService.cs:16]: add empty line

File: SearchMicroservice.Domain/Services/SearchService.cs [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+using RabbitMQConfig.Models.Requests;
+using RabbitMQConfig.Models.Responses;
+using SearchMicroservice.Domain.Interfaces;
+using System.Threading.Tasks;
+
+namespace SearchMicroservice.Domain.Services
+{
+    public class SearchService : ISearchService
+    {
+        private readonly IClientFactory _clientFactory;
```
Comment (SashaKorniichuk) [inline SearchMicroservice.Domain/Services/SearchService.cs:11]: add empty line

File: SearchMicroservice.WebAPI/Controllers/SearchController.cs [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class SearchController : ControllerBase
+    {
+        private readonly ISearchService _searchService;
+        public SearchController(ISearchService searchService)
+        {
+            _searchService = searchService;
+        }
```
Comment (SashaKorniichuk) [inline SearchMicroservice.WebAPI/Controllers/SearchController.cs:16]: add empty line

File: SearchMicroservice.WebAPI/Controllers/SearchController.cs [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+using SearchMicroservice.Domain.Interfaces;
+using System.Threading.Tasks;
+
+namespace SearchMicroservice.WebAPI.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class SearchController : ControllerBase
+    {
+        private readonly ISearchService _searchService;
```
Comment (SashaKorniichuk) [inline SearchMicroservice.WebAPI/Controllers/SearchController.cs:11]: add empty line

**Review by SashaKorniichuk — CHANGES_REQUESTED** [review]

**Review by SashaKorniichuk — APPROVED** [review]

**Review by pinkevmladchy — APPROVED** [review]

**Review by ICHTLAY — APPROVED** [review]

**Pull request merged**